service.py

```diff
@@ -687,6 +687,11 @@ def author(author_id):
     comments = Comment.query.order_by(Comment.time.desc()).filter_by(author_id=author_id).all()
     return render_template('author.html', articles=articles, comments=comments, Tool=Tool)
 
+@app.route('/recent')
+def example():
```

Comment:
@sanoussibarkay 

Please replace the function name `example `with a better name (to reflect the purpose of this function).

**Hui**